config/mcp_config.py

```
@@ -0,0 +1,75 @@
+        
+        claude_config = {
+            "mcpServers": {
+                "self-correcting-executor": {
+                    "command": "python3",
+                    "args": [str(Path(__file__).parent.parent / "mcp_server" / "main.py")]
+                }
+            }
+        }
+        
+        claude_config_path.parent.mkdir(parents=True, exist_ok=True)
+        with open(claude_config_path, 'w') as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)